components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/v1/impl/SystemScopesApiServiceImpl.java

```diff
@@ -28,6 +31,7 @@
 
 public class SystemScopesApiServiceImpl implements SystemScopesApiService {
 
+    private static final Log log = LogFactory.getLog(SystemScopesMappingUtil.class);
```

Comment:
The class should be changed:
```suggestion
    private static final Log log = LogFactory.getLog(SystemScopesApiServiceImpl.class);

```